### PR TITLE
Adds my rope wall blog post

### PR DESCRIPTION
**Pull request by pbranigan**

### CODE REVIEW

File: source/posts/2016-05-04-building-a-space-learning-the-ropes.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+lessons learned from stepping outside our comfort zone and designing for a medium we admittedly are not experts in: our 
+physical work environment. We start with elements of the lobby area.
+
+## Our challenge
+
+Previously when someone entered our office, they’d be greeted at the door and instructed to take a seat. Between them and 
+the immediate seating area was this curved wall. They’d step over it. They’d step around it. They’d even ask us whether to 
+step around it! It was a surprisingly large interruption in the initial experience of visiting the office.
+
+![Floor plan](https://i.imgur.com/cn22Ez8.jpg)
```
Comment (acacheung) [inline source/posts/2016-05-04-building-a-space-learning-the-ropes.md]: may be nice to include where the entrance is in the drawing as well as where the rest of the office is

File: source/posts/2016-05-04-building-a-space-learning-the-ropes.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+These are typical materials needed perhaps when one is exploring creative outlets, studying for a career in the arts, or 
+merely going about their career as a designer. These are our familiar materials when solving a challenge. But how often do 
+you see rope used to solve a design challenge? 
+
+Okay, that’s not a shocking or specific question. Let me rephrase: how often do you see a group of web application designers 
+using rope to solve a spatial design challenge?
+
+![Rope](https://i.imgur.com/fkJyHS4.jpg)
+
+Rope is potent with creative potential because of its malleability. It’s ability to structure complex forms and functions
```
Comment (acacheung) [inline source/posts/2016-05-04-building-a-space-learning-the-ropes.md]: Incorrect It's

File: source/posts/2016-05-04-building-a-space-learning-the-ropes.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+merely going about their career as a designer. These are our familiar materials when solving a challenge. But how often do 
+you see rope used to solve a design challenge? 
+
+Okay, that’s not a shocking or specific question. Let me rephrase: how often do you see a group of web application designers 
+using rope to solve a spatial design challenge?
+
+![Rope](https://i.imgur.com/fkJyHS4.jpg)
+
+Rope is potent with creative potential because of its malleability. It’s ability to structure complex forms and functions 
+was something we believed we could take advantage of. After all, it’s been used since...forever, and in a seemingly infinite
```
Comment (acacheung) [inline source/posts/2016-05-04-building-a-space-learning-the-ropes.md]: "we could use to our advantage"? currently sounds dominating

File: source/posts/2016-05-04-building-a-space-learning-the-ropes.md (hunk 1 above)
Comment (pbranigan) [inline source/posts/2016-05-04-building-a-space-learning-the-ropes.md]: Actually going to omit it because I realize it was in there when I hadn't yet found the image of the actual space before the installation. That's really all that's needed. Thanks!
